ExpensesTracker/Forms/MainWindow.cs

```diff
@@ -27,15 +27,34 @@ public MainWindow()
         private void MainWindow_Load(object sender, EventArgs e)
         {
             _accounts = _storage.GetAccounts();
-            _accountOperations = _storage.GetAccountOperations();
+
+            if (!_accounts.Any())
+                CreateDefaultAccount();
 
             accountBox.DataSource = null;
             accountBox.DataSource = _accounts;
             accountBox.DisplayMember = "Name";
-            accountBox.SelectedIndex = 0;
 
             operationsTable.AutoGenerateColumns = false;
+
+            accountBox.SelectedIndexChanged -= (SelectedAccountBox_SelectedIndexChanged);
+            accountBox.SelectedIndex = 0;
+            accountBox.SelectedIndexChanged += (SelectedAccountBox_SelectedIndexChanged);
+
             categoryFilterBox.SelectedItem = "All amount";
```

Comment:
Remove a subscription from MainWindows.Designer and subscribe after the operation.